src/WatchDogs.Application/Program.cs

```diff
@@ -1,59 +1,76 @@
-
-
-
 using Contracts;
 using Infrastructure.DxTrade;
+using Serilog;
 using System.Net.Http.Headers;
 
-var builder = WebApplication.CreateBuilder(args);
+Log.Logger = new LoggerConfiguration()
```

Comment:
Is the ILogger also added into DI container?

Reply:
When you configure logging with Serilog (or any other logging provider) in the ASP.NET Core dependency injection (DI) container, it automatically registers ILogger<T> for use throughout your application.